api/v1alpha1/cluster_types.go

```diff
@@ -17,10 +17,24 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"github.com/operator-framework/operator-lib/status"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// ConditionInstalling means the tools are installing on your cluster.
+	ConditionInstalling status.ConditionType = "Installing"
```

Comment:
OOC: Are these values required by RH or are we free to customize them?

Reply:
We request some value, but we can set them to whatever we want.